kitsune/users/views.py

```diff
@@ -68,8 +68,26 @@ def user_auth(request, notification=None):
     FXA sign-up/login and the legacy login form
     """
     next_url = get_next_url(request) or reverse("home")
+    terms_url = "https://www.mozilla.org/en-US/about/legal/terms/services/"
+    privacy_url = "https://www.mozilla.org/en-US/privacy/firefox/"
+    help_text = _(
+        "Continuing with Firefox Accounts means you have agreed with the "
+        + "{a_open}"
+        + terms_url
+        + "'>"
+        + "Term and Conditions {a_close} and "
+        + "{a_open}"
+        + privacy_url
+        + "'>"
+        + "Privacy Policy</a>."
+        + "{a_close}"
+    ).format(a_open="<a href='", a_close="</a>")
```

Comment:
This is gonna be a very confusing string to localise: `Continuing with Firefox Accounts means you have agreed with the {a_open}https://www.mozilla.org/en-US/about/legal/terms/services/'>Term and Conditions {a_close} and...`

This should be something like this, so we don't pass any urls or html to localisers:

```python
help_text = _(
    "Continuing with Firefox Accounts means you have agreed with the "
    + "{terms_open}Terms and Conditions{terms_close} and "
    + "{privacy_open}Privacy Policy{privacy_close}."
).format(
    terms_open="<a href='https://www.mozilla.org/about/legal/terms/services/'>",
    terms_close="</a>",
    privacy_open="<a href='https://www.mozilla.org/privacy/firefox/'>",
    privacy_close="</a>",
)
```